js/utils/helpers.js

```diff
@@ -2742,6 +2742,16 @@
         // date object
         obj.$moment = module._currDate;
 
+        // if there is a window object, we are in the browser
+        if (window && window.location) {
+            obj.$location = {
+                origin: window.location.origin,
+                host: window.location.host,
+                hostname: window.location.hostname,
+                chaise_path: window.location.pathname.substring(0, window.location.pathname.indexOf("chaise/") + 7) // +7 to account for "chaise/"
```

Comment:
You should not do this. We went through all the trouble of defining chaise and ermrestjs path as part of build so we don't have to hardcode them.

Using `ermrestjsBuildVariables` you can access the variables that we're populating while building ermrestjs and `chaiseBuildVariables` is the chaise equivalent of that.

I would say use `chaiseBuildVariables.chaiseBasePath` if defined, otherwise just default to `/chaise/`.